manifests/k/KDE/ruqola/1.6.1/KDE.ruqola.installer.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: KDE.ruqola
+PackageVersion: 1.6.1
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: https://binary-factory.kde.org/view/Windows%2064-bit/job/Ruqola_Release_win64/lastSuccessfulBuild/artifact/ruqola-1.6.1-67-windows-msvc2019_64-cl.exe

```

Comment:
```suggestion
  InstallerUrl: https://binary-factory.kde.org/view/Windows%2064-bit/job/Ruqola_Release_win64/67/artifact/ruqola-1.6.1-67-windows-msvc2019_64-cl.exe
```

Reply:
Thanks to the tremendous KDE sysadmins we now have a stable link: https://download.kde.org/stable/ruqola/ruqola-1.6.1-windows-msvc2019_64-cl.exe